README.md

```diff
@@ -25,3 +25,33 @@ cd numba-benchmark
 ```bash
 asv run
 ```
+
+## Comparing two commits
+
+Run `asv` on the first commit:
+
+```bash
+asv run "-1 abcdefg
```

Comment:
```suggestion
asv run "-1 abcdefg"
```
also, that's hexadecimal? Why's there a `g` in it?

Reply:
Not necessarily, it can be any commit-ish - which includes tags and branches. Though I understand that this is misleading because it looks like hex.

Reply:
Fixed in 5c870d33b7649e3bade1a1fe51c55f529a352c8d